src/ripple/test/jtx/Env.h

```diff
@@ -620,6 +628,37 @@ class Env
         AccountID, Account> map_;
 };
 
+//------------------------------------------------------------------------------
+
+namespace detail {
+
+inline
+void
+collect_args (std::vector<std::string>& v)
+{
+}
+
+template <class T, class... Args>
+void
+collect_args (std::vector<std::string>& v,
+    T const& t, Args const&... args)
+{
+    v.emplace_back(t);
+    collect_args(v, args...);
+}
+
+} // detail
+
+template <class T, class... Args>
+std::pair<int, Json::Value>
+Env::rpc (T const& t, Args const&... args)
+{
+    std::vector<std::string> v;
+    detail::collect_args(v, t, args...);
```

Comment:
We should be able to simplify this code by getting rid of `collect_args` and creating the vector from `args...`. Directly. Something like `std::vector<std::string> const v{{std::forward<Args>(args)...}};` (assuming the parameters are passed by forward ref). I made a commit to cherry pick if you'd like: https://github.com/seelabs/rippled/commit/3ee824aae8ffd669e43e88b0e6ee217d857966ed